ibis/bigquery/client.py

```diff
@@ -46,14 +55,30 @@ def __exit__(self, exc_type, exc_value, traceback):
 
 class BigQuery(Query):
 
+    def __init__(self, *args, **kwargs):
+        query_parameters = kwargs.pop('query_parameters', [])
+        super(BigQuery, self).__init__(*args, **kwargs)
+        self.query_parameters = query_parameters
```

Comment:
I might just hate typing but `ibis.bigquery.BigQuery.query_parameters` a bit verbose to me.

Reply:
Users don't interact with ibis at this level, and this is an instance variable anyway so the most you'd ever type is `self.query_parameters`